### ISSUE TITLE
403 Forbidden

### ISSUE THREAD

**Issue by KJMidclark** [DashboardPack/architectui-react-theme-free#57]
Dear Team,

I have tried to get a free theme and when I try to download the theme it says 403 Forbidden on the next page.

If there are any solutions to this please let me know.

Thanks,
Midclark
![Screenshot 2024-02-10 7 07 13 AM](https://github.com/DashboardPack/architectui-react-theme-free/assets/150847333/c6942c0a-0d1f-45fe-927d-31aedb2d042f)

**Comment by Iamshivam-dev** [comment]
Same issue still exists, no luck :(

**Comment by CavaliereDavid** [comment]
I still get the issue, has anyone found a way around?

**Comment by KJMidclark** [comment]
Yeah I don't know yet...

**Comment by puikinsh** [comment]
Just download it here from GitHub

**Comment by CavaliereDavid** [comment]
> Just download it here from GitHub 
Yes I did that in the end. So Is it right to suppose that the download will always be done through github and that there are no future plans to download it from the link?
If so mark the issue as closed.

**Comment by puikinsh** [comment]
Closing — as noted in the thread, the free template is distributed via GitHub. Download the latest release directly from this repository.
